lib/committee/middleware/request_validation.rb

```diff
@@ -16,7 +16,7 @@ def handle(request)
       raise Committee::NotFound, "That request method and path combination isn't defined." if !schema_validator.link_exist? && @strict
 
       @app.call(request.env)
-    rescue Committee::BadRequest, Committee::InvalidRequest
+    rescue Committee::BadRequest, Committee::InvalidRequest, OpenAPIParser::NotExistRequiredKey
```

Comment:
Sorry, this class is a common part so we don't uses OpenAPI or JSON Hyper-Schema specific code.
So we should rescue in Committee::SchemaValidator::OpenAPI3::OperationWrapper class and we need to transfer the error message.

Reply:
When the committee call OpenAPIParser method, rescue error and transfer the error message to committee's error class.
https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/open_api_3/operation_wrapper.rb#L98-L103
https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/open_api_3/operation_wrapper.rb#L110-L112

But we doesn't rescue in path parameter validation.
I think we can solve this problem to rescue error and transfer Committee::InvalidRequest class.
https://github.com/interagent/committee/blob/master/lib/committee/schema_validator/open_api_3/operation_wrapper.rb#L20

Reply:
Tahnks!

I fixed it, And rebased.

Reply:
Thanks!

I fixed it, And rebased.
https://github.com/interagent/committee/pull/225/commits/4308115d6e26156d82720915384b64df422791bc